pkg/probes/curl_json/curl_json.go

```diff
@@ -0,0 +1,83 @@
+package curl_json
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+
+	handledErrors "github.com/openshift/osd-network-verifier/pkg/errors"
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+type CurlJSONProbe struct{}
+
+//go:embed userdata-template.yaml
+var userDataTemplate string
+
+const startingToken = "NV_CURLJSON_BEGIN"
+const endingToken = "NV_CURLJSON_END"
+const outputLinePrefix = "@NV@"
+
+var presetUserDataVariables = map[string]string{
+	"USERDATA_BEGIN": startingToken,
+	"USERDATA_END":   endingToken,
+	"LINE_PREFIX":    outputLinePrefix,
+}
+
+// GetStartingToken returns the string token used to signal the beginning of the probe's output
+func (prb CurlJSONProbe) GetStartingToken() string { return startingToken }
+
+// GetStartingToken returns the string token used to signal the end of the probe's output
```

Comment:
```suggestion
// GetEndingToken returns the string token used to signal the end of the probe's output
```

Reply:
Nice catch, thanks! Fixed in latest commit